src/olympia/shelves/tests/test_views.py

```diff
@@ -100,7 +107,7 @@ def test_only_enabled_shelves_in_view(self):
         self.hpshelf_b.update(enabled=True)
         # don't enable shelf_c
 
-        with self.assertNumQueries(4):
+        with self.assertNumQueries(26):
```

Comment:
😨 22 extra queries?  That's all from a single collection? (with one addon and a sole author)?

Reply:
I just realized the link above that outlined the captured queries did not go to [pastebin](https://pastebin.com/rnq6CzG0), sorry! I was looking through the extra queries, but I wasn't sure which were necessary and/or where it needs to be revised to correct it.

Reply:
coincidentally, this week we started looking at the amazing (in a bad way!) number of queries the collection endpoints make (#16081) so 22 queries doesn't seem so exceptional now.